src/inc/Dwarf.def

```diff
@@ -0,0 +1,393 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Could you please add these files into a dedicate directory (e.g. 'src\inc\llvm')